docs/source/contribution/development_for_databricks.md

```diff
@@ -0,0 +1,89 @@
+* Access to a Databricks workspace with an [existing cluster](https://docs.databricks.com/clusters/create-cluster.html).
+* [GNU `make`](https://www.gnu.org/software/make/).
+* [`git`](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+* A local clone of the [Kedro git repository](https://github.com/kedro-org/kedro).
+
+## How to install a build of Kedro onto Databricks
+
+The development workflow for Kedro on Databricks is similar to the one for Kedro in general, when you develop and test your changes locally. The main difference comes when manually testing your changes on Databricks, since you will need to build and deploy the wheel file to Databricks to test it on a cluster.
+
+To make developing Kedro for Databricks easier, Kedro comes with a `Makefile` target named `databricks-build` that automates the process of building a wheel file and installing this on your Databricks cluster to save development time.
+
+### Setup the Databricks CLI to test a Kedro build
+
+Before you use `make databricks-build`, you must [set up the Databricks CLI](https://docs.databricks.com/dev-tools/cli/index.html#set-up-the-cli).
+
+The only remaining step is to create and environment variable with the ID of the cluster you are using to test your Kedro build. You can find the ID by executing the Databricks CLI command `databricks clusters list` and looking for the Cluster ID to the left of the name of your chosen cluster, for instance:
+
+```bash
+$ databricks clusters list
+1234-567890-abcd1234  General Cluster             TERMINATED
+0987-654321-9876xywz  Kedro Test Cluster          TERMINATED
+```
+
+In this case, the cluster ID of `Kedro Test Cluster` is `0987-654321-9876xywz`.
+
+Once you have determined the cluster ID, you must export it to an environment variable named `DATABRICKS_CLUSTER_ID`:
+
+```bash
+# Linux or macOS
+export DATABRICKS_CLUSTER_ID=<your-cluster-id>
+
+# Windows
+set DATABRICKS_CLUSTER_ID=<your-cluster-id>
+```
+
+### How to use `make databricks-build` to test your Kedro build
+
+With the setup complete, you can use `databricks-build`. In your terminal, navigate to the parent directory of your Kedro development repository and run:
+
+```bash
+make databricks-build
+```
+
+You should see a stream of messages being written to your terminal. Behind the scenes, `build-databricks` does the following:
+
+1. Builds a wheel file of your modified version of Kedro.
+2. Uninstalls any library on your Databricks cluster with the same wheel file name.
+3. Uploads your updated wheel file to DBFS (Databricks File System).
+4. Queues your updated wheel file for installation
+5. Restarts your cluster to apply the changes.
+
+Note that your cluster will be unavailable while it restarts. You can poll the status of the cluster using the Databricks CLI command `databricks clusters get --cluster-id <your-cluster-id> | grep state` (macOS, Linux).
```

Comment:
Still not clear what Windows users do

Reply:
You're right about this, I'm sorry I missed it. I updated both this command and the previous Windows one with PowerShell commands that I verified as working.